pt/news/_posts/2016-10-20-fukuoka-ruby-award-2017.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: news_post
+title: "2017 Fukuoka Ruby Award Competition - Inscrições serão avaliadas por Matz"
+author: "Fukuoka Ruby"
+translator: jcserracampos
+date: 2016-10-20 00:00:00 +0000
+lang: pt
+---
+
+Prezados Entusiastas Ruby,
+
+O Governo de Fukuoka, Japão, juntamente com "Matz" Matsumoto gostaria de lhe
+convidar para se inscrever na seguinte competição Ruby. Se você já desenvolveu um
+programa Ruby interessante, por favor sinta-se encorajado a participar.
+
+2017 Fukuoka Ruby Award Competition - Grande Prêmio - 1 Milhão de Yens!
+
+Prazo de inscrição: 27 de dezembro de 2016
+
+![Fukuoka Ruby Award](http://www.digitalfukuoka.jp/javascripts/kcfinder/upload/images/fukuokarubyaward2017.png)
+
+Matz e um grupo de painelista selecionarão os vencedores da Fukuoka Competition.
+O grande prêmio para a Fukuoka Competition são um milhão de yens.
+Ganhadores do grande prêmio anteriores incluem Rhomobile (EUA) e APEC Climate Center(Coreia)
+
+[http://myfukuoka.com/category/news/ruby-news/](http://myfukuoka.com/category/news/ruby-news/)
+
+Programas inscritos na competição não precisam ser escritos inteiramente em Ruby
+mas devem aproveitar-se de uma das características únicas do Ruby.
```

Comment:
This sentence is correct, I just think it sounds better to write it without `de uma`, like `aproveitar-se das características...`.

With `de uma` it sounds like you're waiting for the unique characteristic. 
Anyway, just pointing that out.